### PR TITLE
Improved ArchiveIt script

### PR DESCRIPTION
**Pull request by shelld3v**

### CODE REVIEW

File: datasrcs/script_exports.go [hunk 1]
```diff
@@ -608,6 +608,17 @@ func (s *Script) scrape(L *lua.LState) int {
 	found = false
 	filter := stringfilter.NewStringFilter()
 	for _, name := range s.subre.FindAllString(resp, -1) {
+		inDomains := false
```
Comment (caffix) [inline datasrcs/script_exports.go:611]: You can use the IsDomainInScope method from the config package to implement this check

**Review by caffix — CHANGES_REQUESTED** [review]
Looks good. Please use the existing routine for checking scope

File: datasrcs/script_exports.go (hunk 1 above)
Comment (shelld3v) [reply datasrcs/script_exports.go:611]: There is a built-in function for this?

Comment (caffix) [reply datasrcs/script_exports.go:611]: Yes. WhichDomain returns the domain in scope that the provided FQDN ends with

Comment (shelld3v) [reply datasrcs/script_exports.go:611]:
```go
if inDomains := cfg.WhichDomain(name); inDomains != "" {
	continue
}
```

Is that correct?

Comment (caffix) [reply datasrcs/script_exports.go:611]:
```go
if domain := cfg.WhichDomain(name); domain == name {
continue
}
```

Comment (shelld3v) [reply datasrcs/script_exports.go:611]: Oh yeah, my bad! I misunderstood the function

**Review by caffix — APPROVED** [review]
Great!

**Pull request merged**